### PR TITLE
[Do not merge] Release 1.5.x

### PR DESCRIPTION
**Pull request by Roman-Kornev**
## PR Checklist:

- [ ] I have implemented all requirements? (see JIRA, project documentation).
- [ ] I am not affecting someone else's work, If I am, they are included as a reviewer.
- [ ] I have added relevant tests (unit, integration or regression).
- [ ] I have added comments to all the bits that are hard to follow.
- [ ] I have added/updated Documentation.
- [ ] I have updated the appropriate changelog with a line for my changes.

### CODE REVIEW

**Comment by krzysztof-causalens** [comment]
@Roman-Kornev If you make another branch from the commit where you branched off and set it as a target it's easier to see what's being released, currently it's targetting master so it will be difficult to see later.
See #60 for how its supposed to be set up

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ importers:
       prettier: 2.4.1
       typescript: ^5.0.4
     dependencies:
-      '@darajs/prettier-config': link:../prettier-config
+      '@darajs/prettier-config': 1.6.0
```
Comment (krzysztof-causalens) [inline pnpm-lock.yaml:37]: Something went wrong here and is using 1.6 instead of 1.5.4

Comment (Roman-Kornev) [reply pnpm-lock.yaml]: Turns out it was picking up an already released 1.6.0 version regardless. Switched from caret range to tilde range

**Comment by Roman-Kornev** [comment]
Switched base for comparison to 1.5.3

**Pull request closed without merging**